### PR TITLE
fix: [CDS-54437]: added/updated tooltip content for service/environme…

### PR DESCRIPTION
**Pull request by sahithikolichala**
…nt and infra dropdowns

### CODE REVIEW

**Review by tan-nhu — APPROVED** [review]

File: src/Tooltip/TooltipDictionary.yaml [hunk 1]
```diff
@@ -147,11 +147,12 @@ deployService_name: |-
     You can also propagate and override a Service in subsequent stages by selecting its name in that stage's **Service** settings. [Learn More](https://developer.harness.io/docs/continuous-delivery/x-platform-cd-features/services/propagate-and-override-cd-services)
 specifyYourEnvironment:
     content: |-
-        Select or create a new Environment. [Learn More](https://developer.harness.io/docs/continuous-delivery/get-started/services-and-environments-overview/)
+        Configure an environment you wish to deploy with this stage. You can define it upfront
+        as a fixed value, pass it in at runtime as a runtime input, or have it computed at runtime as an expression.<br>
 
-        Environments represent your deployment environments (QA, Prod, etc). 
+        <strong>Please note</strong>: when configuring an environment as an expression, Harness will not be able to process any of its dependent runtime inputs in the Studio. The studio doesn't know what the expression (environment identifier) will resolve to.<br>
```
Comment (mtran7) [inline src/Tooltip/TooltipDictionary.yaml:153]: Should capitalize Studio if capitalized in the sentence before. 

Also, have @michaelcretzman or someone on Docs team to look at these statements

File: src/Tooltip/TooltipDictionary.yaml [hunk 2]
```diff
@@ -5502,11 +5503,12 @@ ECSDeploymentTypeManifests:
     width: "400"
 specifyYourInfrastructure:
     content: |-
-        Infrastructure Definitions are the actual clusters, hosts, etc., where Harness deploys a Service.  [Learn More](https://developer.harness.io/docs/continuous-delivery/get-started/services-and-environments-overview/)
+        Configure an infrastructure you wish to deploy at this stage. You can define it upfront
+        as a fixed value, pass it in at runtime as a runtime input, or have it computed at runtime as an expression.</br>
 
-        For example, you might have a QA Environment with separate Infrastructure Definitions for each Kubernetes cluster you use in QA.
+        <strong>Please note</strong>: when configuring an infrastructure as an expression, Harness will not be able to process any of its dependent runtime inputs in the Studio. The studio doesn't know what the expression (infrastructure identifier) will resolve to.<br>
```
Comment (mtran7) [inline src/Tooltip/TooltipDictionary.yaml:5509]: Same comment as above

File: src/Tooltip/TooltipDictionary.yaml [hunk 3]
```diff
@@ -7510,7 +7512,12 @@ cdEditStage_environmentType:
         - **1st gen**: The YAML parameters for Google Functions 1st gen are defined in the [google.cloud.functions.v1 CloudFunction message](https://cloud.google.com/functions/docs/reference/rpc/google.cloud.functions.v1#cloudfunction) from Google Cloud.
     width: "400"
 deployServiceStepForm_service:
-    content: Select or create a Harness service. [Learn More](https://developer.harness.io/docs/continuous-delivery/get-started/services-and-environments-overview/)
+    content: |-
+        Configure a Service you wish to deploy with this stage. You can define it upfront as a fixed value, pass it in at runtime as a runtime input, or have it computed at runtime as an expression.<br>
+
+        <strong>Please note</strong>: when configuring a service as an expression, Harness will not be able to process any of its dependent runtime inputs in the Studio. The studio doesn't know what the expression (service identifier) will resolve to.<br>
```
Comment (mtran7) [inline src/Tooltip/TooltipDictionary.yaml:7518]: Same comment as above

File: src/Tooltip/TooltipDictionary.yaml (hunk 1 above)
Comment (sahithikolichala) [reply src/Tooltip/TooltipDictionary.yaml:153]: yes this message is given by rohan

File: src/Tooltip/TooltipDictionary.yaml [hunk 4]
```diff
@@ -7510,7 +7512,12 @@ cdEditStage_environmentType:
         - **1st gen**: The YAML parameters for Google Functions 1st gen are defined in the [google.cloud.functions.v1 CloudFunction message](https://cloud.google.com/functions/docs/reference/rpc/google.cloud.functions.v1#cloudfunction) from Google Cloud.
     width: "400"
 deployServiceStepForm_service:
-    content: Select or create a Harness service. [Learn More](https://developer.harness.io/docs/continuous-delivery/get-started/services-and-environments-overview/)
+    content: |-
+        Configure a Service you wish to deploy with this stage. You can define it upfront as a fixed value, pass it in at runtime as a runtime input, or have it computed at runtime as an expression.<br>
```
Comment (mtran7) [inline src/Tooltip/TooltipDictionary.yaml:7516]: You have capitalized Configure a "Service" with capital S here but not for the other two instances above for Environment and Infrastructure. Any reason for this?

Comment (sahithikolichala) [reply src/Tooltip/TooltipDictionary.yaml:7516]: I just copy pasted the message from this ticket https://harness.atlassian.net/browse/CDS-54437

Comment (thisrohangupta) [reply src/Tooltip/TooltipDictionary.yaml:7516]: No reason please make the changes.

File: src/Tooltip/TooltipDictionary.yaml [hunk 5]
```diff
@@ -7510,7 +7512,12 @@ cdEditStage_environmentType:
...
+
+        <strong>Please note</strong>: when configuring a service as an expression, Harness will not be able to process any of its dependent runtime inputs in the Studio. The studio doesn't know what the expression (Service Identifier) will resolve to.<br>
```
Comment (mtran7) [inline src/Tooltip/TooltipDictionary.yaml:7518]: Capitalize Studio

File: src/Tooltip/TooltipDictionary.yaml [hunk 6]
```diff
@@ -5502,11 +5503,12 @@ ECSDeploymentTypeManifests:
...
     content: |-
-        Infrastructure Definitions are the actual clusters, hosts, etc., where Harness deploys a Service.  [Learn More](https://developer.harness.io/docs/continuous-delivery/get-started/services-and-environments-overview/)
+        Configure an Infrastructure you wish to deploy at this stage. You can define it upfront
+        as a fixed value, pass it in at runtime as a runtime input, or have it computed at runtime as an expression.</br>
 
-        For example, you might have a QA Environment with separate Infrastructure Definitions for each Kubernetes cluster you use in QA.
+        <strong>Please note</strong>: when configuring an Infrastructure as an expression, Harness will not be able to process any of its dependent runtime inputs in the Studio. The Studio doesn't know what the expression (Infrastructure Identifier) will resolve to.<br>
 
-        You can add multiple Infrastructure Definitions to a single Environment.
+        You can still update the  Pipeline via Pipeline YAML Editor to provide additional inputs.
```
Comment (mtran7) [inline src/Tooltip/TooltipDictionary.yaml:5511]: Extra space before Pipeline

**Review by mtran7 — APPROVED** [review]

**Pull request merged**